src/store/plugins/confirmations.js

```diff
@@ -0,0 +1,33 @@
+import Vue from 'vue';
+import { i18n } from '../../popup/utils/i18nHelper';
+
+const confirmationsText = {
+  login: i18n.t('pages.confirmations.login'),
+  messageSign: i18n.t('pages.confirmations.message-sign'),
+  transactionSign: i18n.t('pages.confirmations.transaction-sign'),
```

Comment:
To manage wording from Vuex looks like overengineering, can't we move it to the component? Also, are you sure that these translations remain reactive?